skycultures/modern/description.en.utf8

```diff
@@ -0,0 +1,91 @@
+Therefore, this sky culture displays a modern descendant of the Ptolemaic one with traces of translations, misunderstandings, transformations, and deliberate additions.</p>
+<p>In 1603 the lawyer and hobby astronomer Johann Bayer from Augsburg, Germany, made a new celestial atlas, called Uranometria [#41]. 
+In this atlas, he introduced a systematic naming of stars starting with alpha, the brightest, beta, the second brightest, gamma, the third ... within a constellation. 
+Subsequently, it was established to name a star with a Greek letter and the Latin constellation name in casus genitive.</p>
+<p>In the 18th century, the century of great atlases (Flamsteed, Fortin, Bode: [#42][#43]), new constellations were added to fill the gaps between northern constellations. 
+Additionally, there were suggestions to create new constellations in the southern part of the sky which had not been visible to Ptolemy. 
+European astronomers visited the European colonies and instead of asking the native people for their constellations, the Europeans were creative. 
+This way, many new technical inventions of the early modern period were depicted in constellations (e.g. a chemical furnace, a balloon, an air pump) 
+as well as exotic animals (e.g. a chamaeleon, a tucan, a paradise bird).</p>
+<p>The problem was that sometimes astronomers did not agree with each other and, thus, atlases showed deviations from each other.</p>
+<p>After many centuries of astronomers taking liberties in interpretation of the Ptolemaic original, there were many variants of constellations. 
+As constellations are a positioning system, i.e. astronomers describe positions in the sky relative to stars and stars relative to constellations 
+(like "Betelgeuse", Orion's right shoulder), it is necessary to use constellation names and star names uniformly all over the world.</p>
+<p>In the "Fin de siècle", there were upcoming suggestions in the astronomical community to unify the system of constellations and star names [#44].</p>
+<p>The IAU, only founded in 1919, in 1928 defined the boundaries of these areas [#43] and not the way of drawing artwork or connecting lines within them. 
+Consequently several sub-cultures of painting figures and drawing simplified stickfigures were developed during the 20th and 21st century. 
+We provide some of them as further sky cultures.</p>
+
+<h3>Constellations</h3>
+<p>These constellations are based mainly on the Ptolemaic tradition which had been used in all three Abrahamitic religions and, 
+thus, was common in the Near East, in Europe and its colonies on all continents. 
+However, between the ancient Greek constellations there were gaps of areas with only faint stars belonging to no constellation.[#47] 
+In the Modern Epoch, several astronomers suggested constellation figures for these gaps 
+(e.g. the constellation of "the lynx" for an area close to the Great Bear but with stars so faint that you would need the eyes of a lynx to see them), 
```

Comment:
shall we really have this sentence twice in the text? @sushoff ?

Reply:
? what do you mean?

Reply:
the line about the lynx constellation being so dim that you need lynx eyes to see it can be read twice within this text. There may be other redundant bits, but other things have distracted my attention these days.